index.html

```diff
@@ -614,6 +618,72 @@ <h4 class="latest">Alibaba shares jump 8% as Jack Ma surfaces after 3 months</h4
       </div>
     </section>
 
+    <!-- LATEST DEALS -->
+
+    <section class="deals">
+      <h2 class='main-h2'>Latest Deals <a href="#"><i class="fas fa-chevron-right"></i></a> </h2>
+
+      <div class="row-deals">
+        <div class="item-deals">
+          <img src="assets/deals1.jpg" alt="deals" class="deals-img">
+          <a href="#">
+            <p class="desc">The Ultimate 2021 White Hat Hacker Certification Bundle</p>
+          </a>
+          <span class="price-1">$1345</span><span class="price-2">$39.90</span>
+        </div>
+        <div class="item-deals">
+          <img src="assets/deals1.jpg" alt="deals" class="deals-img">
+          <a href="#">
+            <p class="desc">The Ultimate 2021 White Hat Hacker Certification Bundle</p>
+          </a>
+          <span class="price-1">$1345</span><span class="price-2">$39.90</span>
+        </div>
+        <div class="item-deals">
+          <img src="assets/deals1.jpg" alt="deals" class="deals-img">
```

Comment:
## `Latest Deals` - Small Screen Change
- [ ] The Change in the `Latest Deals` section requires that the articles get to span the entire page. In the clone page, instead of one, we have two. 👇 Take care of the logo as well. Currently, it does not appear to look as the original logo.

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/MarkoNS1990/the_next_web/milestone-2/index.html)|
|:--:|:--:|
|![latest_deals_small_screen](https://user-images.githubusercontent.com/22345961/109026577-f1fb8580-76d0-11eb-93b5-d577cc2bac6d.png)|![latest_deals_small_screen_clone](https://user-images.githubusercontent.com/22345961/109026596-f627a300-76d0-11eb-808e-d8ebfca3d865.png)|